packaging/darwin/Resources/conclusion.html

```diff
@@ -7,7 +7,7 @@
 <div align="left" style="font-family: Helvetica; padding-left: 10px;">
     <br/>
     <p style="color: #020202; font-size: 12px;">Thanks for installing CodeReady Containers!</p>
-    <p style="color: #020202; font-size: 12px;">You can now open CodeReady Containers from application</b>.</p>
+    <p style="color: #020202; font-size: 12px;">You can now open CodeReady Containers from the Applications</b>.</p>
```

Comment:
maybe drop `the` and just `You can now open CodeReady Containers from Applications` 
cc @robin-owen

Reply:
Is there an official name for this folder?

Reply:
https://support.apple.com/guide/mac-help/folders-that-come-with-your-mac-mchlp1143/mac

`You can now open CodeReady Containers from the Applications folder.`